lua/yabs/util.lua

```diff
@@ -109,4 +109,10 @@ function M.run_command(cmd, output, opts)
     output:run(cmd, opts)
 end
 
+function M.file_exists(file)
+    local f = io.open(file, "rb")
+    if f then f:close() end
+    return f ~= nil
+end
```

Comment:
Maybe it's better to replace the helper with [Plenary.Path.exists()](https://github.com/nvim-lua/plenary.nvim/blob/1c31adb35fcebe921f65e5c6ff6d5481fa5fa5ac/lua/plenary/path.lua#L281)?

Reply:
I'm not sure I want to add plenary as a dependency at this point. I understand most people already have it so that's not a big deal, but I think I'd rather go though and change all the things that could be improved with plenary at once, and I can't get around to doing that right now. It's kinda low priority.

Thanks for the suggestion anyway!

Reply:
Sounds reasonable! Will mark this as resolved (but `vim.loop.cwd()` is still a valid suggestion).

Reply:
What is the difference between these two functions?

Reply:
`vim.fn.*` functions is a vimscript bridge that performs conversion between lua and vimscript types and calls the corresponding vimscript function. It's always better to use "native" functions such as `vim.loop.cwd()` :)

Reply:
Makes sense, got it!